File: 0476-number-complement/0476-number-complement.py
class Solution(object):
    def findComplement(self, num):
        """
        :type num: int
        :rtype: int
        """
        
        value = bin(num)[2:len(bin(num))]
        
        new = ""
        for s in value:
            if s == "0":
                new += "1"
            else:
                new += "0"
        new = "0b"+ new
        
        print(int(new,2))
        return int(new,2)
        

File: 0476-number-complement/0476-number-complement.java
class Solution {
    public int findComplement(int num) {
        String initial = Integer.toBinaryString(num);
        String answer = "";
        for(int i = 0; i < initial.length(); i++){
            if(initial.charAt(i) == '1'){
                answer += "0";
            }
            else{
                answer += "1";
            }
        }
        return Integer.parseInt(answer, 2);
    }
}